extensions/amp-video/0.1/test/test-flexible-bitrate.js

```diff
@@ -78,6 +78,21 @@ describes.fakeWin('amp-video flexible-bitrate', {}, (env) => {
       expect(currentBitrates(v1)[0]).to.equal(1000);
       expect(v1.currentSrc).to.equal('http://localhost:9876/1000.mp4');
     });
+
+    it('should not lower bitrate on loaded video', () => {
+      const m = getManager('4g');
+      const v0 = getVideo([4000, 1000, 3000, 2000]);
+      env.sandbox.stub(v0, 'duration').value(0);
```

Comment:
Can you add one more test that doesn't have a duration of 0?

Reply:
This is definitely a typo, this is supposed to be 10 so the buffer is sufficiently filled 😁